### DISCUSSION TITLE
VMware/Hyper-V VM

### DISCUSSION THREAD

**Discussion by FragRot** [netinvent/npbackup#52] [Q&A]
Do NPBackup support live backup of VMware/Hyper-V VM?

**Comment by deajan** [comment]
NPBackup specialises in file backups.
There's currently a script that supports QEMU/Kvm VM backup, but there are no plans yet to support VMWare / Hyper-V.

As for Hyper-V, if there's enough traction, I could include a script like https://github.com/Digressive/HyperV-Backup-Utility that would work in the same way as the KVM script does, but there would still be some major gap between commercial solutions and this one (CDP, cluster block tracking, live restore to name a few).
